vnext/ReactUWP/EndPoints/dll/react-native-uwp.oss.patched.x64.def

```diff
@@ -0,0 +1,78 @@
+;READFIRST
+;Please keep the ABI surface in sync across all platforms. If you need to add a
```

Comment:
The only difference between this and unpatched is removing a commented out export that we now need for linking